readonly/__init__.py

```diff
@@ -10,8 +10,14 @@
 
 from time import time
 
+import django
 from django.conf import settings
-from django.db.backends import util
+
+if django.VERSION < (1, 7):
```

Comment:
@meastman had a good point. Catching the `ImportError` can mask other import issues that might have occurred. I'll merge as-is.